util/README.md

```diff
@@ -0,0 +1,3 @@
+## util
+
+Testing and bootstrap utilities.
```

Comment:
i think this is not quite right, it also contains like `grpc_util`

Reply:
Can you propose a one sentence descriptor that would be more accurate?

Reply:
miscellaneous utilities

Reply:
there should really be like, at least two categories: serialization/encodings, and networking

Reply:
we dont have to fix it now

Reply:
let's go with misc utils for now